.circleci/config.yml

```diff
@@ -92,6 +92,12 @@ commands:
           name: 'Fail Fast'
           when: on_fail
           command: ./.circleci/fail_fast.sh
+  store_test_output:
+    steps:
+      - store_artifacts:
+          path: result-reports
```

Comment:
I suggest adding the `result-reports` directory to `.gitignore`, `.prettierignore`, and `.eslintignore` so that it's impossible for any of its contents to affect other CI checks.

Reply:
It was already in .gitignore but it looks like it's missing in both `.prettierignore` and `.eslintignore` so I'll add it there.